covalent/executor/executor_plugins/local.py

```diff
@@ -59,6 +63,29 @@ class LocalExecutor(BaseExecutor):
     Local executor class that directly invokes the input function.
     """
 
+    def __init__(
+        self, workdir: str = "", create_unique_workdir: bool = None, *args, **kwargs
```

Comment:
Similarly here i.e. `create_unique_workdir: Optional[bool] = None` where you do `from typing import Optional` at the top. Alternatively, you could also use the default value as `False`